lib/prelude/input.dart

```diff
@@ -1,14 +1,216 @@
-import 'dart:collection';
-
+import 'package:flame/experimental.dart' as experimental;
 import 'package:flame/input.dart';
+import 'package:flutter/gestures.dart';
 import 'package:flutter/services.dart';
 
 /// Resource which contains all the input data for the current frame.
 class Input {
   Set<LogicalKeyboardKey> keysPressed = {};
-  PointerHoverInfo? pointerMove;
-  Vector2 mousePosition = Vector2.zero();
-  HashMap<int, TapDownInfo> taps = HashMap();
-  HashSet<int> tapUps = HashSet();
-  HashSet<int> tapCancels = HashSet();
+  PointerHoverInfo? pointerHover;
+  Vector2 pointerPosition = Vector2.zero();
+
+  // Taps
+  List<experimental.TapDownEvent> taps = [];
```

Comment:
Might mistaken, but the dart way would be making the list finels?

Reply:
Here unsure, because I need to clear them - currently do that by setting an empty list, but there might be a `clear(...)` method

Reply:
See the comment later one, there is a clear method